src/app/screens/Register/i18n.ts

```diff
@@ -0,0 +1,31 @@
+import i18next from 'i18next';
+
+i18next.addResources('es', 'Register', {
+  logoAlt: 'logo Wolox',
+  firstName: 'Nombre',
+  lastName: 'Apellido',
+  email: 'Correo',
```

Comment:
i think some of those will be reused on login screen. could be moved to a more general level